src/k8s-engine/wsl.ts

```diff
@@ -1198,6 +1229,22 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
     // No implementation warnings available.
   }
 
+  // Set up the background process for integrating with a different WSL
+  // distribution to proxy the dockerd socket.
+  protected async setupIntegrationProcess(distro: string) {
+    const executable = await this.getWSLHelperPath();
+
+    this.integrationProcesses[distro] ??= new BackgroundProcess(this, `${ distro } socket proxy`, async() => {
+      return childProcess.spawn('wsl.exe',
+        ['--distribution', distro, '--user', 'root', '--exec', executable, 'docker-proxy', 'serve'],
```

Comment:
I want to use `uid` instead of user name here, but `'--user', '0` just resulted in WSL complaining about unknown user.

Reply:
Sounds fine. User 0 has been called `root` for over 50 years, and `wsl` doesn't take a UserID argument.